solution/graph_traversal/2638/main.py

```diff
@@ -0,0 +1,52 @@
+# Authored by : seonjoo0885
+# Co-authored by : -
+# Link : http://boj.kr/0dfe13fd2bf04a1e850ae2707c79076f
+
+import sys
+from collections import deque
+def input():
+    return sys.stdin.readline().rstrip()
+
+def bfs(x, y):
+    dx = [-1, 1, 0, 0]
+    dy = [0, 0, -1, 1]
+    air = deque([(x, y)])
+    visited = [[0] * m for _ in range(n)]
+    melt = set()
+    while air:
+        x, y = air.popleft()
+        for i in range(4):
+            nx = x + dx[i]
+            ny = y + dy[i]
+            if 0 <= nx < n and 0 <= ny < m and not visited[nx][ny]:
+                if cheeze[nx][ny]:  # 치즈라면
+                    melt.add((nx, ny))
+                    cheeze[nx][ny] += 1
+                elif not cheeze[nx][ny]:  # 공기라면
+                    air.append((nx, ny))
+                    visited[nx][ny] = 1
+    meltCnt = 0
```

Comment:
nitpick) python에서의 변수명은 기본적으로 snake case입니다.
https://peps.python.org/pep-0008/#function-and-variable-names
```suggestion
    melt_cnt = 0
```